### PR TITLE
Implement new links functionality

### PR DESCRIPTION
**Pull request by costebano**

### CODE REVIEW

File: UI/src/app/components/pages/mapping/mapping.component.scss [hunk 1]
```diff
@@ -1,65 +1,72 @@
...
+    height: 100%;
+    position: relative;
+    float: left;
+  }
 
-        }
+  .hint {
+    position: absolute;
+    left: 35%;
+    color: #404040;
```
Comment (etokareva) [inline UI/src/app/components/pages/mapping/mapping.component.scss:27]: Please use $cdm-grey-dark from variables

File: UI/src/app/components/pages/mapping/mapping.component.ts [hunk 2]
```diff
@@ -53,12 +56,88 @@ export class MappingComponent extends BaseComponent implements OnInit, OnDestroy
...
+    private overlayService: OverlayService
   ) {
     super();
     this.commonService.mappingElement = mappingElementRef;
   }
 
+  ngAfterViewInit() {
+    this.svgCanvas.nativeElement.addEventListener('mouseup', (event: any) => {
+      const areaWidth = 16;
+      const offsetX = event.offsetX;
```
Comment (etokareva) [inline UI/src/app/components/pages/mapping/mapping.component.ts:69]: use ES6 destructuring here

File: UI/src/app/models/arrow.ts [hunk 3]
```diff
@@ -110,33 +110,52 @@ export class Arrow implements IConnector {
...
   }
 
   attachButton(button) {
     this.button = button;
   }
 
+  setEndMarkerType(type: string): void {
+    this.refreshPathHtmlElement();
+
+    const isActive = this.svgPath.attributes[9].value.includes('active');
```
Comment (etokareva) [inline UI/src/app/models/arrow.ts:123]: 9 is magic number there. It's not obvious why we use it

Comment (costebano) [reply UI/src/app/models/arrow.ts:123]: This is the index of the item in attributes. I can move it to constant. Will it be enough?

File: UI/src/app/components/pages/mapping/mapping.component.scss (hunk 1 above)
Comment (costebano) [reply UI/src/app/components/pages/mapping/mapping.component.scss]: fixed

File: UI/src/app/models/arrow.ts (hunk 3 above)
Comment (costebano) [reply UI/src/app/models/arrow.ts]: moved

File: UI/src/app/components/pages/mapping/mapping.component.ts (hunk 2 above)
Comment (costebano) [reply UI/src/app/components/pages/mapping/mapping.component.ts]: done

**Pull request merged**